### PR TITLE
Update user guide

### PR DESCRIPTION
**Pull request by lungsin**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -56,85 +66,128 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
...
+* Shows the statistics of the cards with the given tag. If no tags are given, statistics of all cards will be shown.
+****
+
+Examples:
+
+* `stats t/physics`
+
 
-=== Selecting a person : `select`
+=== [Coming in version 2.0] Selecting a person : `select`
```
Comment (sergiovieri) [inline docs/UserGuide.adoc]: @lungsin Make this and others "[Coming in v2.0]"? (to make it short)

Comment (lungsin) [reply docs/UserGuide.adoc]: Updated

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -12,18 +12,28 @@ ifdef::env-github[]
...
+experience for more experienced users.
+
+Interested? Jump to the <<Quick Start>> to get started. Enjoy!
 
 == Quick Start
 
 .  Ensure you have Java version `9` or later installed in your Computer.
-.  Download the latest `addressbook.jar` link:{repoURL}/releases[here].
+.  Download the latest `acquizition.jar` link:{repoURL}/releases[here].
 .  Copy the file to the folder you want to use as the home folder for your Address Book.
```
Comment (sergiovieri) [inline docs/UserGuide.adoc]: Some things are still addressbook. This and the command summary below.

**Review by sergiovieri — CHANGES_REQUESTED** [review]

File: docs/UserGuide.adoc (hunk 2 above)
Comment (lungsin) [reply docs/UserGuide.adoc]: ah, missed that one. fixed.

**Review by sergiovieri — APPROVED** [review]

**Pull request merged**